src/runner/index.ts

```diff
@@ -51,9 +51,18 @@ export default async function run(
       env.HUSKY_GIT_STDIN = await getStdinFn()
     }
 
-    if (command) {
-      console.log(`husky > ${hookName} (node ${process.version})`)
-      execa.shellSync(command, { cwd, env, stdio: 'inherit' })
+    if (commands) {
+      if (!Array.isArray(commands)) {
+        commands = [commands]
+      }
+
+      commands.forEach((command: string) => {
+        console.log(
+          `husky > ${hookName} > ${command} (node ${process.version})`
+        )
+        execa.shellSync(command, { cwd, env, stdio: 'inherit' })
```

Comment:
What if a command fails?

Reply:
Yeah, you're right.
I'll change that if the PR will be relevant.